azurerm/resource_arm_app_service_plan.go

```diff
@@ -136,12 +139,16 @@ func resourceArmAppServicePlan() *schema.Resource {
 				ConflictsWith: []string{"properties.0.reserved"},
 			},
 
-			"maximum_number_of_workers": {
+			"maximum_elastic_worker_count": {
```

Comment:
Could we add some validation to this property?

Reply:
I'm not sure what type of validation you want to add to an Int here?

Reply:
(chatting offline, we're adding `IntAtLeast: 0` here)

Reply:
ta 👍